modules/statics/src/coins/ofcErc20Coins.ts

```diff
@@ -609,6 +609,14 @@ export const ofcErc20Coins = [
   ofcerc20('81d817b5-da65-41aa-983a-ec9619a6a46f', 'ofcedr', 'Endor Protocol', 18, underlyingAssetForSymbol('edr')),
   ofcerc20('c74b96eb-2b24-43a6-a718-936526651f3a', 'ofcefi', 'Efinity', 18, underlyingAssetForSymbol('efi')),
   ofcerc20('e925c83f-bcbd-42a6-90f0-40cc8eae54bd', 'ofcegold', 'eGold', 4, underlyingAssetForSymbol('egold')),
+  ofcerc20(
+    '03423ec5-a65f-428d-9a96-1affb41d583e',
+    'eth:ecash',
```

Comment:
this should be `ofceth:ecash`
```suggestion
    'ofceth:ecash',
```

Reply:
Addressed your comments — please take another look.